### PR TITLE
Fix checkpoint loader to use GFile

### PR DESCRIPTION
**Pull request by kchygoe**
## What this patch does to fix the issue.
When `OUTPUT_DIR=gs://xxxx/xxx`,
Training cannot be kept running.
This is critical for gcs/s3 users

### CODE REVIEW

File: blueoil/cmd/train.py [hunk 1]
```diff
@@ -395,8 +395,8 @@ def train(config_file, experiment_id=None, recreate=False):
     if not tf.io.gfile.exists(checkpoint):
         raise Exception('Checkpoints are not created in {}'.format(experiment_dir))
 
-    with open(checkpoint) as stream:
-        data = yaml.load(stream)
+    with tf.io.gfile.GFile(checkpoint) as stream:
+        data = yaml.load(stream, Loader=yaml.FullLoader)
```
Comment (tsawada) [inline blueoil/cmd/train.py]: is this loader change related to GFile change?

It seems there are a few other use of `yaml.load` within blueoil thus maybe we should fix all in another diff.

Comment (kchygoe) [reply blueoil/cmd/train.py]: no, this is not directly related.

I just noticed that `load` method has been changed to specify `Loader` . 
Let me dedicate them as you commented.
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

Comment (kchygoe) [reply blueoil/cmd/train.py]: fixed
